state/protocol/badger/mutator_test.go

```diff
@@ -1365,7 +1505,7 @@ func TestEmergencyEpochFallback(t *testing.T) {
 			err = state.Finalize(context.Background(), block1.ID())
 			require.NoError(t, err)
 
-			assertEpochEmergencyFallbackTriggered(t, state)
+			assertEpochEmergencyFallbackTriggered(t, state, true)
```

Comment:
I really liked how you have checked that it wasn't triggered before and after some action(finalizing block) it triggers one, can we also add an assert that it wasn't triggered before?

Reply:
https://github.com/onflow/flow-go/pull/2969/commits/7909a51d8e111a58d62f60346d8b6efaedc0cc2f 👍